src/app/app.component.html

```diff
@@ -1,3 +1,9 @@
 <app-header></app-header>
-<app-name-list></app-name-list>
-<app-about></app-about>
+<mat-grid-list cols="3" gutterSize="15px" rowHeight="fit">
+  <mat-grid-tile colspan="2">
```

Comment:
Having issue here with scrolling. Viewport is smaller than the content and I can get a scroll bar to show up but can't scroll all the way back to the top, so can't see the filter buttons.